libs/util/app-config/src/lib/app-config.ts

```diff
@@ -70,14 +82,14 @@ export function loadAppConfig() {
     .then((conf) => {
       let parsed
       try {
-        parsed = TOML.parse(conf, { joiner: '\n' }) as any
+        parsed = TOML.parse(conf, { joiner: '\n', bigint: false }) as any
```

Comment:
why did you need this ?

Reply:
Otherwise, I got `max_zoom = 10` in config file parsed as `max_zoom: 10n`